pkg/network/ebpf/c/protocols/http2/decoding.h

```diff
@@ -248,11 +255,18 @@ static __always_inline void process_headers(struct __sk_buff *skb, dynamic_table
             current_stream->is_huffman_encoded = dynamic_value->is_huffman_encoded;
             bpf_memcpy(current_stream->request_path, dynamic_value->buffer, HTTP2_MAX_PATH_LEN);
         } else {
+            // We're in new dynamic header or new dynamic header not indexed states.
             dynamic_value.string_len = current_header->new_dynamic_value_size;
             dynamic_value.is_huffman_encoded = current_header->is_huffman_encoded;
```

Comment:
should be deleted as you added it in the `if (current_header->type == kNewDynamicHeader)`